### PR TITLE
bump kerlescan to 0.55

### PR DESCRIPTION
**Pull request by eherget**
## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [x] Input Validation
- [x] Output Encoding
- [x] Authentication and Password Management
- [x] Session Management
- [x] Access Control
- [x] Cryptographic Practices
- [x] Error Handling and Logging
- [x] Data Protection
- [x] Communication Security
- [x] System Configuration
- [x] Database Security
- [x] File Management
- [x] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -17,6 +16,7 @@ psycopg2 = "*"
 logstash-formatter = "*"
 python-dateutil = "*"
 pytz = "*"
+kerlescan = {editable = true, ref = "0.55", git = "https://github.com/RedHatInsights/kerlescan.git"}
```
Comment (sthirugn) [inline Pipfile:19]: any reason for the following changes?
- the line order from line 6 to 19? (it was alphabetical order before)
- the order of these fields (editable, ref, git) is also changed, previously it was alphabetical order (editable, git, ref) previously

**Review by sthirugn — CHANGES_REQUESTED** [review]

**Comment by sthirugn** [comment]
> ## Secure Coding Practices Checklist Link
> 
>     * https://github.com/RedHatInsights/secure-coding-checklist
> 
> 
> ## Secure Coding Checklist
> 
>     * [ ]  Input Validation
> 
>     * [ ]  Output Encoding
> 
>     * [ ]  Authentication and Password Management
> 
>     * [ ]  Session Management
> 
>     * [ ]  Access Control
> 
>     * [ ]  Cryptographic Practices
> 
>     * [ ]  Error Handling and Logging
> 
>     * [ ]  Data Protection
> 
>     * [ ]  Communication Security
> 
>     * [ ]  System Configuration
> 
>     * [ ]  Database Security
> 
>     * [ ]  File Management
> 
>     * [ ]  Memory Management
> 
>     * [ ]  General Coding Practices

I think you are supposed to check these items in the checklist to make sure that you reviewed them when you submitted the PR.

File: Pipfile (hunk 1 above)
Comment (eherget) [reply Pipfile]: I used the `pipenv uninstall kerlescan` command, followed by `pipenv install --editable git+https://github.com/RedHatInsights/kerlescan.git@0.55#egg=kerlescan` command to get pipenv to install the correct version of kerlescan into my env.  Without doing this, regardless of running `pipenv update`, `pipenv lock`, `pipenv install --dev`, etc, running the unit tests would complain about not finding the kerlescan module.
The `pipenv install ...` command put the args in this order and the entry in the position to where it was moved.

I have updated the order so kerlescan is back in alphbetical order with the other modules.  In system-baseline-backend the modules were not in alphabetical order to begin with.

**Comment by eherget** [comment]
> > ## Secure Coding Practices Checklist Link
> > ```
> > * https://github.com/RedHatInsights/secure-coding-checklist
> > ```
> > 
> > 
> > ## Secure Coding Checklist
> > ```
> > * [ ]  Input Validation
> > 
> > * [ ]  Output Encoding
> > 
> > * [ ]  Authentication and Password Management
> > 
> > * [ ]  Session Management
> > 
> > * [ ]  Access Control
> > 
> > * [ ]  Cryptographic Practices
> > 
> > * [ ]  Error Handling and Logging
> > 
> > * [ ]  Data Protection
> > 
> > * [ ]  Communication Security
> > 
> > * [ ]  System Configuration
> > 
> > * [ ]  Database Security
> > 
> > * [ ]  File Management
> > 
> > * [ ]  Memory Management
> > 
> > * [ ]  General Coding Practices
> > ```
> 
> I think you are supposed to check these items in the checklist to make sure that you reviewed them when you submitted the PR.

I think we are, too.  But this was just a version change of a dependency.  There was no code in the PR.  Plus I'm trying to get back to focusing on FedRAMP stuff and this detour keeps getting longer!

**Comment by sthirugn** [comment]
> I think we are, too. But this was just a version change of a dependency. There was no code in the PR. Plus I'm trying to get back to focusing on FedRAMP stuff and this detour keeps getting longer!

I went ahead and checked all the items on the checklist.

File: Pipfile (hunk 1 above)
Comment (sthirugn) [reply Pipfile]: Thanks

**Review by CodeHeeler — APPROVED** [review]

**Review by sthirugn — APPROVED** [review]

**Pull request merged**